components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -91,6 +103,10 @@ - (void)setRightView:(UIView *)rightView {
   [self mdc_setRightView:rightView];
 }
 
+- (CGRect)clearButtonRectForBounds:(CGRect)bounds {
+  return CGRectZero;
```

Comment:
Please document why you're doing this here.

Reply:
Done.